fast-dump/src/main/java/com/squareup/leakcanary/FastDump.kt

```diff
@@ -0,0 +1,16 @@
+package com.squareup.leakcanary
+
+import androidx.annotation.Keep
+
+@Keep
+object FastDump {
+  init {
+    try {
+      System.loadLibrary("fast-dump")
+    } catch (e: UnsatisfiedLinkError ) {
+      e.printStackTrace()
```

Comment:
Ideally there should probably an explicit load method, maybe with a boolean indicating whether loading was successfull. Then the consumer can check that before calling forkDumpHprof

Reply:
That would make the reflection calls a bit more cumbersome, but also allows to fallback to default if it didn't work.

Reply:
The check & fallback implements in `DumpWrapper.dumpHprofData` with InvocationTargetException